azure-pipelines.yml

```diff
@@ -42,11 +42,11 @@ jobs:
           $(EXTRA_CMAKE_ARGS)
           $(Build.SourcesDirectory)
 
-    - task: CMake@1
-      displayName: "Check code formatting"
-      inputs:
-        workingDirectory: $(Build.BinariesDirectory)/build
-        cmakeArgs: --build . --target format_check
+    #- task: CMake@1
```

Comment:
Why?

Reply:
Everything I changed in the Azure pipelines file is temporary; I'll restore the original one in the final commit